fw-child/resources/functions/builder/front-end/block/content/social-links.php

```diff
@@ -71,6 +84,7 @@
 						target="_blank"
 						rel="noopener"
 						title="<?php echo esc_attr( $link[ 'label' ] ) ?>"
+						style="<?php echo ( ! empty( $link[ 'brand-color' ] ) ) ? "color: {$link[ 'brand-color' ]};" : "" ?>"
```

Comment:
I would put the whole style attr in the condition to avoid echoing an empty style attr.
I would remove the extra parentheses around the condition. (WCS: No Extra Parentheses)
I would remove the extra spaces inside the square brackets.